### PR TITLE
Add Tizen Studio 3.0 Release Notes

### PR DESCRIPTION
**Pull request by adityaaswani**
Add Tizen Studio 3.0 Release Notes

### CODE REVIEW

**Comment by mijin85cho** [comment]
Hello, 
I assigned @surbhig12 to this PR instead of @annie-samsung, as @annie-samsung is absent for now.

@surbhig12, please assign a reviewer to this PR.

@adityaaswani, to add a new page, 
please edit the docs/application/tizen-studio/TOC.md file and docs/application/tizen-studio/release-notes/release-notes.md also.

I think the link https://github.com/Samsung/tizen-docs/pull/349/files can be an example to add a release note.

File: docs/application/tizen-studio/release-notes/3-0-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+-	Certificate Manager
+	-   Overwriting a duplicate certificate profile in the migration wizard works incorrectly on macOS.
+-	Native UI Builder
+	-   If the expanded attribute in a multibutton entry component is set to false, + is displayed.
+-	Native Component Designer
+	-   The vector-type part is not supported. You cannot see the vector image and change the SVG file.
+	-   Playing sound is not supported on Windows&reg; or macOS.
+	-   The Component Designer crashes if an alias is selected as the source group of an added item.
+-	Emulator
+	-   To use the Tizen emulator, install an Intel VTx supported by the CPU, and the latest version of the graphic card driver provided by the vendor. To verify the prerequisites for the Tizen emulator, see [Prerequisites for the Tizen Studio](https://developer.tizen.org/development/tizen-studio/download/installing-tizen-studio/prerequisites).
```
Comment (mijin85cho) [inline docs/application/tizen-studio/release-notes/3-0-release-notes.md]: Hello, @Reni-90.
If possible please guide to use the link by md files in tizen-docs/ repository. 
As `../setup/prerequisites.md`.

File: docs/application/tizen-studio/release-notes/3-0-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	    -   If the RAM size is set to 768 or 1024 MB for the VM in the Emulator Manager, change it to 512 MB.
+	    -   Increase the user area of the virtual memory in the system to 3 GB by entering the *bcdedit /setincreaseuserva 3072* command on the console with administrator rights (only in Windows&reg; 7), and reboot.
+	-   If you use a MacBook Pro which has both Intel HD and NVIDIA&reg; GPUs, the emulator can unexpectedly terminate when you execute the emulator with OpenGL ES version 1.1 or 2.0. Verify the emulator configuration in the Emulator Manager and on the general tab in the emulator configuration window, set OpenGL ES version to version 2.0 or to version 3.0.
+	-   When you launch the Emulator Manager in the Tizen IDE, the shortcut image of Emulator Manager may not be displayed properly.
+	-   Basic Web applications does not install on SD cards.
+-	CLI and SDB
+	-   The Tizen Studio does not support the [Smart Development Bridge](https://developer.tizen.org/development/tizen-studio/web-tools/running-and-testing-your-app/sdb)(SDB) bash auto-completion on Windows&reg; (it is available on Ubuntu and macOS).
```
Comment (mijin85cho) [inline docs/application/tizen-studio/release-notes/3-0-release-notes.md]: Hello, @Reni-90.
If possible please guide to use the link by md files in tizen-docs/ repository. 
As `../common-tools/smart-development-bridge.md`.

File: docs/application/tizen-studio/release-notes/3-0-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+-	Tizen Studio
+	-	In case of Mac, if the UI perspectives are not displayed properly after updating the Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from previous installation.
+	-	In case if you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+-	Common
+	-   If you install the Tizen Studio in a directory that requires administrator privileges for access. For example, for C:\ProgramFiles, administrator privileges are required to run the Tizen SDK tools. The Tizen Installer and the baseline SDK Installer alerts you, if you try to install into such a directory.
+-	Web and Native IDE
+	-   From Tizen Studio 2.0 onwards, the Connection Explorer is replaced with the Device Manager, this can cause errors in the Connection Explorer view. You can fix this in two ways:
+	    -   Reset the perspective.
+            In the Tizen Studio menu, select Window > Perspective > Reset Perspective
+	    -   After updating to the Tizen Studio 2.0, run the *eclipse.exe -clean -clearPersistedState* command. Then launch the Tizen Studio normally.
```
Comment (mijin85cho) [inline docs/application/tizen-studio/release-notes/3-0-release-notes.md]: Global. Please avoid using Italic. Please use bold instead.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/428

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/428

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/428

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/428

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/428

**Comment by mijin85cho** [comment]
Thanks for the review and update.

**Comment by mijin85cho** [comment]
Even this PR is worked on master branch,
as the target date is Oct 31, 
the Tizen Developer Site will show this update on Oct 31.

**Pull request merged**